aspnetcore/blazor/hybrid/tutorials/maui.md

```diff
@@ -0,0 +1,111 @@
+* [Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/): WebView2 is required on Windows when running a native app. When developing .NET MAUI Blazor apps and only running them in Visual Studio's emulators, WebView2 isn't required.
+
+## Create a .NET MAUI Blazor app
+
+Launch Visual Studio 2022 Preview. In the start window, select **Create a new project**:
+
+:::image type="content" source="maui/_static/new-solution.png" alt-text="New solution.":::
+
+In the **Create a new project** window, use the **Project type** drop-down to filter **MAUI** templates:
+
+:::image type="content" source="maui/_static/new-project-1.png" alt-text="Filter templates to .NET MAUI.":::
+
+Select the **.NET MAUI Blazor App (Preview)** template and then select the **Next** button:
+
+:::image type="content" source="maui/_static/new-project-2.png" alt-text="Choose a template.":::
+
+In the **Configure your new project** window, name your project, choose a suitable location for it, and select the **Create** button:
+
+:::image type="content" source="maui/_static/configure-project.png" alt-text="Configure the project.":::
+
+Wait for Visual Studio to create the project and for the project's dependencies to be restored:
+
+:::image type="content" source="maui/_static/restored-dependencies.png" alt-text="Restored dependencies.":::
+
+In the **Android SDK License Acceptance** window, select the **Accept** button:
+
+:::image type="content" source="maui/_static/android-sdk-license.png" alt-text="Android SDK License Acceptance window.":::
+
+Wait for Visual Studio to download the Android SDK and Android Emulator.
+
+## Run the app in the Android Emulator
+
+In the Visual Studio toolbar, select the **Android Emulator** button to build the app:
+
+:::image type="content" source="maui/_static/android-emulator-button.png" alt-text="Android Emulator button.":::
+
+In the **New Device** window, select the **Create** button:
+
+:::image type="content" source="maui/_static/new-android-device.png" alt-text="New Android Device window.":::
+
+In the **License Acceptance** window, select the **Accept** button:
+
+:::image type="content" source="maui/_static/license-acceptance.png" alt-text="License Acceptance window.":::
+
+Wait for Visual Studio to download, unzip, and create an Android Emulator.
+
+Close the **Android Device Manager** window.
+
+In the Visual Studio toolbar, select the **Pixel 5 - {VERSION}** button to build and run the app, where the `{VERSION}` placeholder is the Android version. In the following example, the Android version is `API 30 (Android 11.0 - API 30)`, and a later version appears depending on the Android SDK installed:
+
+:::image type="content" source="maui/_static/pixel5-api30.png" alt-text="Pixel 5 API 30 emulator button.":::
+
+Visual Studio starts the Android Emulator, builds the app, and deploys the app to the emulator.
```

Comment:
Let's add the screenshot of the app running in the Android emulator at the end of this section.